### ISSUE TITLE
fix: `windowsActivate()` not reliable

### ISSUE THREAD

**Issue by cmorten** [guidepup/guidepup#45]
`windowsActivate()` is frequently failing to activate the application window so that it takes focus allowing for screen-reader control.

Relates to https://github.com/guidepup/setup/issues/15

**Comment by cmorten** [comment]
> @cmorten did you ever make progress on this? I want to kick the tires on automating NVDA, and curious how much of a blocker this is for actually running a suite.

Not particularly… it seems by design Windows OS limits your ability to force focus to an application.

Workarounds thus far have been to use the NVDA itself (or similar) to navigate to the target application, perhaps more realistic though not necessarily quick.

**Comment by ckundo** [comment]
thanks for that! looking forward to trying it out more. great work!

**Comment by cmorten** [comment]
Closing as this isn't something that can't be worked around as part of the OS.
